juspay/hyperswitch
# feat(payouts): extend routing capabilities to payout operation

## Type of Change



## Description
Explained in issue [#1845](https://github.com/juspay/hyperswitch-cloud/issues/1845)


### Additional Changes





## Motivation and Context



## How did you test it?

Tested locally using Postman collection for payouts routing - https://postman.co/workspace/My-Workspace~2b563e0d-bad3-420f-8c0b-0fd5b278a4fe/collection/9906252-a04b7f48-7689-4eac-8a84-9b5c5af8ba50?action=share&creator=9906252

**Steps for testing**
_Assuming merchant account and payout connectors are created._

All these steps can be found in the linked postman collection.

1. Add a volume split rule
2. Activate this rule
3. Create multiple payouts (and check the connectors being evaluated in the response)
4. Deactivate this rule
5. Add an advanced split rule
6. Create multiple payouts (and check the connectors being evaluated in the response))
7. Activate this rule
8. Deactivate this rule


## Checklist
